packages/pixeloven-webpack/config/src/helpers/shared.ts

```diff
@@ -376,7 +377,11 @@ export function getSetup(options: Options) {
     }
 
     function getNode() {
-        return ifClient(
+        return ifNode(
```

Comment:
Might not be part of the fix but needed since this is more accurate of what we should be doing.